joomla/joomla-cms
# Use new JLanguageHelper::getInstalledLanguages across core

### Summary of Changes

- Replace all `JLanguageMultilang::getSiteLangs()` for  `JLanguageHelper::getInstalledLanguages(0)` and deprecate the old method.
- Use this new method to replace some hardcoded language db queries, removing duplicated code

### Testing Instructions

1. Code review
2. Use latest staging and apply patch
3. All installed language (selected list in install, backend login and user lang config), language redirect (language switcher module and language filter plugin), etc works as before.
4. Test also disabling installed site/admin languages in Extensions -> Manage -> Manage (Languages) and check all works properly

### Documentation Changes Required

None.
